components/molecules/WorkspaceRepositoryInsightRow/workspace-repository-insight-row.tsx

```diff
@@ -0,0 +1,138 @@
+import Link from "next/link";
+
+import { BsPencilFill } from "react-icons/bs";
+import { MdOutlineArrowForwardIos } from "react-icons/md";
+import { User } from "@supabase/supabase-js";
+
+import clsx from "clsx";
+import { getRelativeDays } from "lib/utils/date-utils";
+import useRepositories from "lib/hooks/api/useRepositories";
+import getRepoInsights from "lib/utils/get-repo-insights";
+import Text from "components/atoms/Typography/text";
+import Pill from "components/atoms/Pill/pill";
+import Card from "components/atoms/Card/card";
+import useInsightRepositories from "lib/hooks/useInsightRepositories";
+import { useWorkspaceMembers } from "lib/hooks/api/useWorkspaceMembers";
+import getPercent from "lib/utils/get-percent";
+
+import CardRepoList from "../CardRepoList/card-repo-list";
+
+interface WorkspaceRepositoryInsightRowProps {
+  insight: DbWorkspaceRepositoryInsight;
+  user: User | null;
+  isEditable?: boolean;
+  workspaceId: string | null;
+}
+
+const WorkspaceRepositoryInsightRow = ({
```

Comment:
This is a copy of the `InsightRow` component that is workspace-aware of repositories and members